### ISSUE TITLE
Unable to claim BCH from Block.io multisig address

### ISSUE THREAD

**Issue by Sem6000** [fyookball/electrum#369]
I have attempted to recover my BCH from my multisig account. 

I have the following details:
`

        Address: 35KaBnrDq2BDNBBSXbSVGpWaY65KZdtJkb
        Address Type: P2SH
        Required Signatures: 2

`

I have the redeem script, the redeem script hex, the public keys and the private keys but I will not be posting those obvoiusly.

When I try to put my details into Electron Cash, I cannot click "next" as seen in this screenshot: https://prnt.sc/hnqul5

**Comment by yeastmeister** [comment]
Had the same problem.  Created the initial wallet with default method.  After that, told electrum to create a new wallet.  The new wallet let me put in the address and click next.

**Comment by Thrash370** [comment]
Having the same problem  For the life of me can't figure out how to get block.io to import to Electron. 

Tried making a new wallet that doesn't work.

Do I have to put xpub or anything like that before the address?  It asks for a public key and private key at same time, how do I separate those (ctrl+enter)?

Any help would be great have a decent amount of BCH I'm trying to get unlocked for a friend.

**Comment by kyuupichan** [comment]
Electron Cash and Electrum only handle multisig HD wallets.  You need a master public key (or master private key).  Those begin with xpub and xprv.  Do you have those?

**Comment by Thrash370** [comment]
It gives me 2 private keys and 2 public keys but doesn't say xpub or xprv.  Please tell me there is some hope to get the bch out!  Very worried

**Comment by kyuupichan** [comment]
So it doesn't come with a seed or anything like that, this wallet?  It just creates 2 random private keys?

Your funds are accessible but would require bespoke code; Electron Cash doesn't have such code.

**Comment by Thrash370** [comment]
I get a redeem script:2 and a redeem script hex not sure if any of that is relevant.

**Comment by Thrash370** [comment]
I don't know if it helps at all but would be willing to pay a bounty of some sort to resolve this

**Comment by vincentrinot** [comment]
I too am having the same issue recovering from an electrum wallet where electron cash doesnt let me press next after entering the private key... I've got the recovery seed and the private key (i have all of them, but i have the specific one where i can see my BCH on the blockchain) . Ive disabled 2FA on electrum and saved the wallet file. Upon opening that wallet file with electron cash, the program crashes (not sure if worth noting but the electrum wallet is being saved on ubuntu and opened on mac os in electron cash). I've also tried opening the wallet from other services and sweeping it, no luck.

**Comment by Sem6000** [comment]
If you want to just pull the funds out, then here is a guide:

First of all you should get from security settings of you block.io profile: "Redeem Script Hex", and two private keys. (actually you will have some more strings and values but only these three required to claim your BCH).

After that:

Go to http://www.motelmaya.com/BCH/BCH.html

Make sure to remove your BTC from your block.io wallet before this.

So, then:

    Click +TX or New -> Transaction. Enter the redeem HEX that you received from block.io in the first box (Address, WIF key or Redeem Script)
    Click on "Advanced options" -> "Network" -> Settings and change the network to "Bitcoin Cash (mainnet)"
    Click New -> Transaction once more.
    Put in the amount you want to transfer and the address you are sending to (your new BCH wallet), make sure that the fee is not too high (I've entered something like 0.0001)
    Click "Submit", you will get a long HEX string. copy it
    Click "Sign", paste the copied string, paste the first private key from the block.io to the "Private key" field, click "Submit", copy the string.
    Paste the string into the second box (below the "Private key" field), paste the second private key from the block.io, click Submit, copy the output
    Go to "Broadcast", paste the string and click "Submit".

That's it. You will recieve TXid which you will be able to check at https://bitcoincash.blockexplorer.com


If you have found this tutorial useful, please tip:
BCH: 12qDszgvkvviKDyYsZ15BJSpgMfwpUqKqv
BTC:  18zKzhdKdhjFXrTRgMQDSJY8j3iNQ9yXg1
ETH: 0xd71d8b168e5d307ba1260ddac456ea33f9d3ffef
VTC: VmTprhWtYuTHmUvC4NCNgZ8pH1xC6Xcibs
DOGE: DPEGgjGP1LcQJqvmdeCawyL9opBrKpGhQo

**Comment by fyookball** [comment]
doesnt sound like an Electron Cash bug.

**Comment by vincentrinot** [comment]
Coinomi support told me since the address starts with "3Dayz"...
particularly the "3", that it's most likely a segwit address and that's why
I'm unable to get access to the wallet.

**Comment by Sem6000** [comment]
The instructions I posted above should work for segwit addresses. However if you cannot figure it out, email me at semmart6@gmail.com

**Comment by vincentrinot** [comment]
Hello Fyookball,
     Hope things are good on your end. You and I have emailed back and
forth earlier in the year about helping me access my bitcoin cash.

A quick recap, I made and moved funds to an electrum wallet shortly before
the bch fork and then moved it back to another wallet. I've been unable to
recover funds with the recovery seed or the private key. I am able to see
the funds on the block chain explorer via the public key.

I recently found the recovery seed from the first time I wrote it down and
see there's a password with it, perhaps this is why I've been unable to
recovery it? I'm not sure where an option to enter a password would be when
I have a private key handy but it's a new piece of information once
forgotten.

I appreciate you taking the time to help me, I know this seems like a
simple issue and it should be. I'm far from computer illiterate and know
what I'm trying to do and what should happen but it's just not doing that.

Thanks in advance!
-Vincent
